config/admins.txt

```diff
@@ -133,3 +133,4 @@ Plizzard = Game Master
 octareenroon91 = Game Master
 Serpentarium = Game Master
 Averagejoe82 = Game Master
+The Dreamweaver = Game Master
```

Comment:
sneaky beaky

Reply:
@Xaltonon This is for coding purpose and the ability to utilize the test server. It isn't used by the game servers. It's quite common for coders to add their name to it, I just haven't got around to doing it until now.

Reply:
is joke

Reply:
Finally, a QoL for Maintainers.